utils.py

```diff
@@ -0,0 +1,43 @@
+import jwt
+
+from django.http             import JsonResponse
+
+from nocchaloc.settings      import SECRET_KEY
+from users.models            import User
+from datetime                import datetime, timedelta
+
+def encode_jwt(user_id):
+    token = jwt.encode(
+        {'user_id':user_id, 
+        'exp':datetime.utcnow() + timedelta(days=1)}, 
+        SECRET_KEY, 
+        algorithm='HS256',
+    )
+    return token
+
+def authorization(func):
+    def wrapper(self, request, *args, **kwargs):
+        
+        token = request.headers.get('Authorization', None)
+        
+        if token == None:
+            return JsonResponse({'error':'Enter the token'}, status=401)
```

Comment:
```suggestion
        if token is None:
            return JsonResponse({'error':'Enter the token'}, status=401)
```
None 값은 보통 Is로 비교합니다.

Reply:
수정했습니닷!